src/romaine/parser/__init__.py

```diff
@@ -0,0 +1,34 @@
+from romaine.parser import (
+    simple,
+    multiline,
+    step,
+    section,
+    feature,
+    )
+from romaine.parser.exceptions import (  # noqa
+    UnclosedPythonishString,
+    MalformedTableError,
+    )
+
+
+class Parser(object):
+    """
+        Gherkin parser for romaine core.
```

Comment:
(minor) These docstrings are overindented (should line up with start of triple-quotes)